docs/reference/nuget-config-file.md

```diff
@@ -125,6 +125,9 @@ Lists all known package sources. The order is ignored during restore operations
 </packageSources>
 ```
 
+> [!NOTE]
```

Comment:
I'm hesitant to add this, mostly because we're not really document `--source`, so why should the msbuild version take precedence?

Reply:
I had the same thoughts but decided that a note would be okay for the time being. My other choice was to not document at all. I went with the PR instead.

Reply:
Perhaps I can add --source CLI too?

Reply:
I think we should add all or none. 

Link to the dotnet.exe, nuget.exe and msbuild.exe equivalents.